guides/v2.3/graphql/tutorials/checkout/checkout-payment-method.md

```diff
@@ -54,6 +54,16 @@ query {
 }
 ```
 
+There are two mutation queries in GraphQl which can be use to set the payment method for your order:
+
+|Mutation|Description|
+|--- |--- |
+|`setPaymentMethodOnCart`|Sets the payment method for your order|
+|`setPaymentMethodAndPlaceOrder`|Sets the payment method and then immediately places your order. In this case ["Step 10. Place the order"]({{ page.baseurl }}/graphql/tutorials/checkout/checkout-place-order.md) can be skipped|
```

Comment:
```suggestion
|`setPaymentMethodAndPlaceOrder`|Sets the payment method and then immediately places your order. In this case ["Step 10. Place the order"]({{ page.baseurl }}/graphql/tutorials/checkout/checkout-place-order.html) can be skipped|
```

Reply:
Thank you, @keharper

Reply:
fixed by @keharper ;)

Reply:
@atwixfirster I can't commit the change. I can only make a suggestion to fix it

Reply:
Not sure how but you fixed that bug. I've pulled the changes and detected what `md` extension has been replaced with `html`.

Please run tests now.

Thank you!